app/public/css/styles.css

```diff
@@ -1830,6 +1903,45 @@ button[data-balloon] {
   margin-bottom: 64px;
 }
 
+/* REWARDS */
+
+.rewards-wrapper {
+  border-top: 1px solid var(--color-border);
+  margin: 18px 0 0 0;
+}
+
+.rewards-title {
+  margin: 6px 0 0 0;
+}
+
+.rewards-row {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.rewards-amount {
+  font-size: 18px;
+  line-height: 48px;
+  color: var(--color-brand);
+  padding-right: 18px;
+  position: relative;
+}
+
+.rewards-amount::after {
+  content: '';
+  position: absolute;
+  width: 11px;
+  height: 15px;
+  right: 0;
+  top: 16px;
+  margin: auto;
+  background-image: url('../assets/ada_symbol_balance.svg');
+  background-repeat: no-repeat;
+  background-size: contain;
+  background-position: center;
```

Comment:
I know it's implemented like this in other parts of code, but let's avoid putting svgs as background images in the future. Try to change it the same way you changed ledger icons - if it won't be too much trouble. Same goes for other delegations css.

Reply:
Moved both Balance and Rewards ADA icons from pseudo-element backgrounds into svgs.